src/AutoMapper/ConfigurationStore.cs

```diff
@@ -364,6 +364,17 @@ public TypeMap FindTypeMapFor(TypePair typePair)
             return typeMap;
         }
 
+        private bool CoveredByObjectMap(TypePair typePair)
+        {
+            return GetMappers().FirstOrDefault(m => m.IsMatch(new ResolutionContext(null, null, typePair.SourceType, typePair.DestinationType, new MappingOperationOptions(), new MappingEngine(this)))) != null;
+        }

```

Comment:
This bothers me, is there not a better way to see if something is covered by an object map? Do we need to check _all_ mappers?